src/Core/View.js

```diff
@@ -79,6 +87,44 @@ function View(crs, viewerDiv, options = {}) {
     if (__DEBUG__) {
         this.isDebugMode = true;
     }
+
+    if (options.loadingScreen !== false) {
```

Comment:
loadingScreen has to be outsourced

Reply:
Can you explain what you mean please?

Reply:
the loading screen should gone in the examples

Reply:
I don't think so.
A default loading screen doesn't cost us much and provide a nicer default experience (see https://github.com/iTowns/itowns/pull/636#issuecomment-363471759 for a more elaborate answer).

Reply:
Or move it to a GUI library.
it's better to separate the html interface and the core

Reply:
> Or move it to a GUI library.

No. Because the point is to have something built-in so it works by default.

>  it's better to separate the html interface and the core

This is a rule of thumb, not an absolute rule that works in all case. 
In this specific case it doesn't make sense because again, I believe that a nicer experience by default is important and the cost (20 lines of JS) is negligible.